src/components/@shared/Publisher/index.tsx

```diff
@@ -41,13 +38,6 @@ export default function Publisher({
         setAccountEns(accountEns)
         setName(accountEns)
       }
-
-      // 3box
-      const profile = await get3BoxProfile(account, source.token)
```

Comment:
removing this effectively removes all 3box support for the publisher line, also for those for which it still works. When looking at current live (v3) market, 3box is still working for all the accounts which have highest liquidity assets:

<img width="1133" alt="Screen Shot 2022-04-04 at 13 41 14" src="https://user-images.githubusercontent.com/90316/161545717-a253dd99-db91-446a-a9d9-9ef1584dfe01.png">

Migrating 3box to its successor, or something else, should be left for #695 and for now we leave this in place also for publisher line here

Reply:
you're right, 3box restored 👍🏽 thanks for that